Dockerfile

```diff
@@ -1,20 +1,9 @@
-FROM python:3.8.5-slim-buster
-
-ARG TOKEN
-
-WORKDIR /dmibot
-
-RUN apt-get update && \
-	apt-get install -y
+FROM python:latest
+WORKDIR /dmibot/
 
 COPY requirements.txt .
-#Install requirements
 RUN pip3 install -r requirements.txt
 
 COPY . .
-#Final setup settings and databases
-RUN mv data/DMI_DB.db.dist data/DMI_DB.db &&\
-	mv config/settings.yaml.dist config/settings.yaml &&\
-	python3 setup.py ${TOKEN}
 
-CMD ["python3", "main.py"]
\ No newline at end of file
+ENTRYPOINT ["python3", "main.py"]
```

Comment:
when python4 will be released this will not work anymore because we are using python:latest and not a specified python3 version, I would revert this.

i prefer the previous version of this dockerfile

Reply:
Ok so we can use python:3 image.